immutable-collection-encoding/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hubspot.immutables</groupId>
+    <artifactId>hubspot-immutables</artifactId>
+    <version>1.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>immutable-collection-encoding</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <basepom.check.skip-checkstyle>true</basepom.check.skip-checkstyle>
```

Comment:
Is it possible to not skip this? It's nice to always have this enabled on new modules